pyls/config/pylint_conf.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2019 Palantir Technologies, Inc.
+import logging
+import os
+from pyls._utils import find_parents
+from .source import ConfigSource, _get_opt, _set_opt
+
+log = logging.getLogger(__name__)
+
+PROJECT_CONFIGS = ['.pylintrc', 'pylintrc']
```

Comment:
So we are only using the first 2 options?

https://docs.pylint.org/en/1.6.0/run.html

```
pylintrc in the current working directory
.pylintrc in the current working directory
If the current working directory is in a Python module, Pylint searches up the hierarchy of Python modules until it finds a pylintrc file. This allows you to specify coding standards on a module-by-module basis. Of course, a directory is judged to be a Python module if it contains an __init__.py file.
The file named by environment variable PYLINTRC
if you have a home directory which isn’t /root:
.pylintrc in your home directory
.config/pylintrc in your home directory
/etc/pylintrc
```